src/main/java/com/dnsimple/endpoints/Registrar.java

```diff
@@ -43,8 +43,9 @@ public Registrar(HttpEndpointClient client) {
      * @param action     The action to get the price of
      * @return The premium price
      * @see <a href="https://developer.dnsimple.com/v2/registrar/#getDomainPremiumPrice">https://developer.dnsimple.com/v2/registrar/#getDomainPremiumPrice</a>
+     * @deprecated As of this version 0.9.1, replaced by {@link #getDomainPrices(Number, String)}
      */
-    public SimpleResponse<DomainPremiumPriceCheck> getDomainPremiumPrice(Number account, String domainName, DomainCheckPremiumPriceAction action) {
+    @Deprecated public SimpleResponse<DomainPremiumPriceCheck> getDomainPremiumPrice(Number account, String domainName, DomainCheckPremiumPriceAction action) {
```

Comment:
```suggestion
    @Deprecated 
    public SimpleResponse<DomainPremiumPriceCheck> getDomainPremiumPrice(Number account, String domainName, DomainCheckPremiumPriceAction action) {
```
Just a silly formatting change